config-gen/apis/v1alpha1/testdata/componentconfig/expected.yaml

```diff
@@ -0,0 +1,323 @@
+apiVersion: v1
```

Comment:
could not the testdata be generated in the testdata/ dir and checked in the CI.. WDYT?